.github/actions/commit-generated-files/action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Commit generated files'
+description: 'Automatically commit generated files'
+runs:
+  using: composite
+  steps:
+    - run: |
+        if [ ! -z "$(git status --porcelain)" ]; then  
+          git fetch --depth=1
+          git checkout ${{ github.head_ref }}
+          git add 'packages/atomic/**/*.d.ts' 'packages/atomic-react/src/components/stencil-generated/*'
+
+          git config --global user.name "GitHub Actions Bot"
+          git config --global user.email "<>"
```

Comment:
Is this git configuration happening elsewhere? If so, can it be centralized in one place?

Reply:
I just copied what is done for bump version on master.

What is in that config is not really important, just needs to be evident to a dev looking at their PR that "some github bot did commit some stuff for me".